.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Ignore all of these files in the .idea folder, they're created by an IDE and are unrelated to the project.